rna_transcription.rb

```diff
@@ -0,0 +1,26 @@
+module BookKeeping
+  VERSION = 4
+end
+class Complement
+  def self.of_dna(dna)
+    dna.upcase!
+    rna = ''
+    dna = dna.split("")
+    (0...(dna.length)).each do |i|
+      case dna[i]
```

Comment:
if i do this way then how i can **_assign empty_** in else block....to the rna???